modules/ztp-creating-siteconfig-custom-resources.adoc

```diff
@@ -137,21 +137,23 @@ spec:
     name: <cluster-image-set> <1>
   networking:
     clusterNetwork:
-    - cidr: 10.128.0.0/14
+    - cidr: <cluster-network-cidr> <2>
       hostPrefix: 23
     machineNetwork:
-    - cidr: <machine-network-cidr> <2>
+    - cidr: <machine-network-cidr> <3>
     serviceNetwork:
-    - 172.30.0.0/16
+    - <service-network-cidr> <4>
   provisionRequirements:
     controlPlaneAgents: 1
     workerAgents: 0
-  sshPublicKey: <public-key> <3>
+  sshPublicKey: <public-key> <5>
 ----
 +
 <1> `cluster-image-set` is the name of the ClusterImageSet custom resource used to install {product-title} on the DU.
-<2> `machine-network-cidr` is the target bare metal machine’s CIDR.
-<3> `public-key` entered as plain text can be used to SSH into the target bare metal machine after the host is installed.
+<2> `cluster-network-cidr` is a block of IPv4 or IPv6 addresses in CIRD notation used for communication among cluster nodes. 
```

Comment:
small typo: s/CIRD notation/CIDR notation. Update throughout.

Reply:
Thank you! I am... a bit dyslexic with acronyms :_)